### PR TITLE
Game class PrintOptions, MakeAction

### LINKED ISSUE #41: PrintOptions()

### LINKED ISSUE #44: MakeAction(Action choice)

### PR DESCRIPTION
**Pull request by lisie-sprawy**
Closed #41 #44

### CODE REVIEW

File: shm/Game.cpp [hunk 1]
```diff
@@ -9,4 +10,31 @@ bool Game::checkWinCondition() const {
...
\ No newline at end of file
+}
+
+void Game::PrintOptions() {
+    std::cout << "Ahoy captain! We're waiting for your commands! \n";
+    std::cout << "0. Exit \n";
+    std::cout << "1. Travel \n";
+    std::cout << "2. Buy \n";
+    std::cout << "3. Sell \n";
+}
```
Comment (iprice73) [inline shm/Game.cpp]: I think it's better to user to start from 1 and put exit as a last option

**Review by iprice73 — APPROVED** [review]

File: shm/Game.cpp (hunk 1 above)
Comment (J-Cieplinski) [reply shm/Game.cpp]: I concur. Exit should be the last one

**Review by J-Cieplinski — CHANGES_REQUESTED** [review]

File: shm/Game.hpp [hunk 2]
```diff
@@ -4,6 +4,16 @@
...
 class Game {
+public:
+    Game(size_t money, size_t days, size_t final_goal);
+
+    enum class Action {
+        Exit,
+        Travel,
+        Buy,
+        Sell,
+    };
```
Comment (niemarcin) [inline shm/Game.hpp]: You can place it outside of the class definition.

File: shm/Game.hpp [hunk 3]
```diff
@@ -14,13 +24,10 @@ class Game {
...
-        sell,
-        travel,
-    };
+    void PrintOptions();
+    void MakeAction(Action choice);
+    void Exit();
+    void Travel();
+    void Buy();
+    void Sell();
 };
```
Comment (niemarcin) [inline shm/Game.hpp]: No empty line at the end of file.

**Review by niemarcin — CHANGES_REQUESTED** [review]

File: shm/Game.cpp (hunk 1 above)
Comment (lisie-sprawy) [reply shm/Game.cpp]: thx:)

File: shm/Game.cpp [hunk 4]
```diff
@@ -14,10 +14,10 @@ bool Game::checkLoseCondition() const {
 
 void Game::PrintOptions() {
     std::cout << "Ahoy captain! We're waiting for your commands! \n";
-    std::cout << "0. Exit \n";
     std::cout << "1. Travel \n";
     std::cout << "2. Buy \n";
     std::cout << "3. Sell \n";
+    std::cout << "0. Exit \n";
```
Comment (J-Cieplinski) [inline shm/Game.cpp:20]: Let's just go with 4 instead of 0.

**Review by J-Cieplinski — CHANGES_REQUESTED** [review]

File: shm/Game.cpp (hunk 4 above)
Comment (ArturGubala) [reply shm/Game.cpp:20]: I think that 0 for exit it's a good idea. If we want to implement another options to menu, like shipyard to sell/buy ships, each time we will move exit to next number. It isn't a big deal to change number for exit, but 0 will be more durable.

**Review by ArturGubala — APPROVED** [review]

File: shm/Game.cpp (hunk 4 above)
Comment (lisie-sprawy) [reply shm/Game.cpp:20]: I have the same way of thinking like Artur:)

**Review by J-Cieplinski — APPROVED** [review]

**Review by niemarcin — APPROVED** [review]

**Pull request merged**